node/flatpak-node-generator.py

```diff
@@ -488,6 +488,39 @@ class Package(NamedTuple):
     lockfile: Path
 
 
+class NodeHeaders:
+    def __init__(self, target: str, runtime: str = None, disturl: str = None):
+        self.target = target
+        if runtime is not None:
+            self.runtime = runtime
+        else:
+            self.runtime = 'node'
```

Comment:
maybe missing something obvious but, why not just `runtime: str = 'node'`?

Reply:
Because we can't pass `runtime=None` to fallback to `node` otherwise.

Reply:
Is there anywhere where that's done though? Looking through this PR now, I see two places where NodeHeaders is used, neither of which `runtime` can be `None`. I'd rather generally avoid propagating `None`s for now if we can avoid it...

Reply:
It comes from rcfile and could be `None`, here:
https://github.com/flatpak/flatpak-builder-tools/blob/9c43ee9196f94d35390d654ac78915cd8bf3bd6b/node/flatpak-node-generator.py#L1394-L1396